[![Docker Repository on Quay](https://quay.io/repository/broadinstitute/viral-phylo/status "Docker Repository on Quay")](https://quay.io/repository/broadinstitute/viral-phylo)
[![Build Status](https://travis-ci.com/broadinstitute/viral-phylo.svg?branch=master)](https://travis-ci.com/broadinstitute/viral-phylo)
[![Documentation Status](https://readthedocs.org/projects/viral-phylo/badge/?version=latest)](https://viral-phylo.readthedocs.io/en/latest/?badge=latest)
<!--
[![Coverage Status](https://coveralls.io/repos/broadinstitute/viral-phylo/badge.png)](https://coveralls.io/r/broadinstitute/viral-phylo)
[![Code Health](https://landscape.io/github/broadinstitute/viral-phylo/master/landscape.svg?style=flat)](https://landscape.io/github/broadinstitute/viral-phylo)
-->

viral-phylo
===========

A set of scripts and tools for the analysis of viral genomes: phylogenetics, annotation, intrahost variants, etc.

More detailed command line documentation can be found at [readthedocs](http://viral-phylo.readthedocs.org/)

Developer documentation can be found at [here](DEVELOPMENT_NOTES.md)

Higher level pipelines and workflows for viral-ngs can be found in the [pipelines github repo](https://github.com/broadinstitute/viral-pipelines).
